test/helpers/setup.js

```diff
@@ -27,6 +27,24 @@ async function setup(options) {
         await appendFile(cleanupFile, `${name};${id}\n`, { encoding: 'utf-8' });
     }
 
+    const allScopes = [
+        'user:read',
+        'user:write',
+        'auth:read',
+        'auth:write',
+        'chart:read',
+        'chart:write',
+        'team:read',
+        'team:write',
+        'folder:read',
+        'folder:write',
+        'plugin:read',
+        'plugin:write',
+        'theme:read',
+        'product:read',
+        'visualization'
```

Comment:
```suggestion
        'visualization:read'
```

Should we change this for consistency?

Reply:
absolutely